test/JDBC/expected/BABEL-3643.out

```diff
@@ -0,0 +1,25 @@
+sp_helpdb 'MaSTeR'
+go
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar#!#int#!#nvarchar#!#nvarchar#!#tinyint
+master#!#<NULL>#!#jdbc_user#!#1#!#2023-03-08 #!#<NULL>#!#<NULL>
+~~END~~
+
+~~START~~
+nchar#!#smallint#!#nchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#varchar
+<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>#!#<NULL>
+~~END~~
+
+
+sp_helpdb 'master'
+go
+~~START~~
+nvarchar#!#nvarchar#!#nvarchar#!#int#!#nvarchar#!#nvarchar#!#tinyint
+master#!#<NULL>#!#jdbc_user#!#1#!#2023-03-08 #!#<NULL>#!#<NULL>
```

Comment:
sp_helpdb could not be tested like this, since it contains non-static information (timestamp). 
In general, all expected results should be consistent across run. 
Please refer to existing tests (I guess only tests the internal helper function and only select db name would be enough for your case)